raven_python_lambda/__init__.py

```diff
@@ -96,7 +109,7 @@ def __init__(self, config=None):
             'filter_local': boolval(os.environ.get('SENTRY_FILTER_LOCAL', True)),
             'is_local': os.environ.get('IS_OFFLINE', False) or os.environ.get('IS_LOCAL', False),
             'logging': boolval(os.environ.get('SENTRY_CAPTURE_LOGS', True)),
-            'log_level': int(os.environ.get('SENTRY_LOG_LEVEL', logging.WARNING)),
+            'log_level': extract_log_level_from_environment('SENTRY_LOG_LEVEL', logging.WARNING),
```

Comment:
Simpler way which doesnt require a new function:
```python
log_levels.get(os.environ.get('SENTRY_LOG_LEVEL'), logging.WARNING)
```

Reply:
That would work for the new use case but doesn't preserve backwards compatibility (setting `SENTRY_LOG_LEVEL` to a numeric value).

Reply:
Agreed -- LGTM.